app/services/facets/configuration.rb

```diff
@@ -0,0 +1,31 @@
+module Facets
+  class Configuration
```

Comment:
This logic can go in `Facets`, and then `facets_registry` can be called just `registry` as it would be called from other classes as `Facets.registry`